src/linear_algebra/affine_space.lean

```diff
@@ -250,6 +270,41 @@ def mk' (f : P1 → P2) (f' : V1 →ₗ[k] V2) (p : P1) (h : ∀ p' : P1, f p' =
 
 @[simp] lemma mk'_linear (f : P1 → P2) (f' : V1 →ₗ[k] V2) (p h) : (mk' f f' p h).linear = f' := rfl
 
+/-- The set of affine maps to a vector space is an additive commutative group. -/
+instance : add_comm_group (affine_map k V1 P1 V2 V2) :=
+{ zero := ⟨0, 0, λ p v, (add_action.zero_vadd _ _).symm⟩,
+  add := λ f g, ⟨f + g, f.linear + g.linear, λ p v, by simp [add_add_add_comm]⟩,
+  neg := λ f, ⟨-f, -f.linear, λ p v, by simp [add_comm]⟩,
+  add_assoc := λ f₁ f₂ f₃, ext $ λ p, add_assoc _ _ _,
+  zero_add := λ f, ext $ λ p, zero_add (f p),
+  add_zero := λ f, ext $ λ p, add_zero (f p),
+  add_comm := λ f g, ext $ λ p, add_comm (f p) (g p),
+  add_left_neg := λ f, ext $ λ p, add_left_neg (f p) }
+
+@[simp, norm_cast] lemma coe_zero : ⇑(0 : affine_map k V1 P1 V2 V2) = 0 := rfl
+@[simp] lemma zero_linear : (0 : affine_map k V1 P1 V2 V2).linear = 0 := rfl
+@[simp, norm_cast] lemma coe_add (f g : affine_map k V1 P1 V2 V2) : ⇑(f + g) = f + g := rfl
+@[simp]
+lemma add_linear (f g : affine_map k V1 P1 V2 V2) : (f + g).linear = f.linear + g.linear := rfl
+
+instance : affine_space k (affine_map k V1 P1 V2 V2) (affine_map k V1 P1 V2 P2) :=
```

Comment:
A docstring would probably help readers noticing that `V2 V2` thing ensuring that one of the affine spaces involved is actually a linear space.

Reply:
Done.